application/controllers/survey_response.py

```diff
@@ -34,10 +34,16 @@ def add_survey_response(self, case_id, file, file_name, survey_ref):
 
         tx_id = str(uuid.uuid4())
         log.info('Adding survey response file', filename=file_name, case_id=case_id, survey_id=survey_ref, tx_id=tx_id)
-
         file_contents = file.read()
-        json_message = self._create_json_message_for_file(file_name, file_contents, case_id, survey_ref)
-        return send_message_to_rabbitmq_queue(json_message, tx_id, RABBIT_QUEUE_NAME)
+        file_size = len(file_contents)
+
+        if self.check_if_file_size_too_small(file_size):
+            log.info('File size is too small')
+            return 'False'
```

Comment:
I'm not sure about returning false as a string here, it should be the boolean type false.

I think it would be better to return a boolean and a string from this function. The boolean to indicate if it was successful and a string relating to the error message